app/src/main/java/hous/release/android/presentation/profile/ProfileFragment.kt

```diff
@@ -40,28 +41,40 @@ class ProfileFragment : BindingFragment<FragmentProfileBinding>(R.layout.fragmen
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         binding.vm = profileViewModel
-        initProfileInfo()
+        initUiStateCollect()
         initNotificationOnClickListener()
         initSettingOnClickListener()
         initBadgeOnClickListener()
         initEditOnClickListener()
         initTestBtnOnClickListener()
         initPersonalityAdapter()
         initPersonalityOnClickListener()
-        observePersonalityPentagon()
     }
 
-    private fun initPersonalityOnClickListener() {
-        binding.llProfilePersonalityDetail.setOnClickListener {
-            clickDateLogEvent(CLICK_MY_PERSONALITY)
-            val toPersonalityResult =
-                Intent(requireActivity(), PersonalityResultActivity::class.java)
-            toPersonalityResult.putExtra(
-                RESULT_COLOR,
-                profileViewModel.profileData.value!!.personalityColor.name
-            )
-            toPersonalityResult.putExtra(LOCATION, PROFILE)
-            startActivity(toPersonalityResult)
+    private fun initUiStateCollect() {
+        repeatOnStarted {
+            profileViewModel.uiState.collect { uiState ->
+                if (uiState.profile.introduction.isNullOrEmpty()) {
+                    with(binding.tvProfileIntroduction) {
+                        setText(R.string.profile_empty_introduction)
+                        setTextColor(ContextCompat.getColor(requireContext(), R.color.hous_g_4))
+                    }
+                } else {
+                    with(binding.tvProfileIntroduction) {
+                        text = uiState.profile.introduction
+                        setTextColor(ContextCompat.getColor(requireContext(), R.color.hous_g_6))
+                    }
+                }
```

Comment:
```suggestion
                with(binding) { 
                  if (uiState.profile.introduction.isNullOrEmpty()) {
                        tvProfileIntroduction.setText(R.string.profile_empty_introduction)
                        tvProfileIntroduction.setTextColor(ContextCompat.getColor(requireContext(), R.color.hous_g_4))
                  } else {
                        tvProfileIntroduction.text = uiState.profile.introduction
                        tvProfileIntroduction.setTextColor(ContextCompat.getColor(requireContext(), R.color.hous_g_6))
                  }
                }
```
음 스니펫이 이상하긴 한데 이렇게 if 문 위에다가 with를 1번만 사용할 수 있을 것 같습니다.